### PR TITLE
Log important events emitted by LightStep tracer

### PR DESCRIPTION
**Pull request by sarslanhan**
The recommended way to log information regarding issues is to use `SetGlobalEventHandler` (see [this](https://github.com/lightstep/lightstep-tracer-go/blob/master/options.go#L165) and the default logger logs only the first error event (see [this](https://github.com/lightstep/lightstep-tracer-go/blob/master/event_handlers.go#L72)).

I actually wanted to inject `logging.Logger`on [`InitTracer`](https://github.com/zalando/skipper/blob/master/tracing/tracing.go#L68). However, this would break the behavior described [here](https://github.com/zalando/skipper/blob/e20191f29c77e0eb8487356545f65d699ae75bf6/docs/reference/plugins.md#opentracing-plugins). Let me know you prefer to do this another way.

### CODE REVIEW

File: tracing/tracers/lightstep/lightstep.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package lightstep
 import (
 	"errors"
 	"fmt"
+	"github.com/zalando/skipper/logging"
```
Comment (szuecs) [inline tracing/tracers/lightstep/lightstep.go]: I think you should use plain github.com/sirupsen/logrus, instead.

Comment (sarslanhan) [reply tracing/tracers/lightstep/lightstep.go]: Sure, I was not sure which one as it is used in a mixed way in different parts of the code. Will send an update.

**Comment by szuecs** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**